### ISSUE TITLE
after the first plot it no longer uses all the threads in the first phase

### ISSUE THREAD

**Issue by marbvelous** [madMAx43v3r/chia-plotter#340]
After creating the first plot I have to end the powershell and restart the pc or it does not use all the processor threads in the first phase and increases the creation time by twenty%. Does anyone else happen to you? thanks

**Comment by lordzamura** [comment]
no. if you something like that it means you have a problem.

**Comment by marbvelous** [comment]
Maybe the hpool miner can be the problem?

**Comment by whitetechnologies** [comment]
@marbvelous No, I can confirm that running hpool miner with plotter is working fine. I only have issues with plotter when r > 16 and then it crahes. Obviously you can just test by not running hpool miner and see if multiple plots with maxmax work fine. Also I have not tested Win10, only Ubuntu at this time so worst case you can load that up and test it (if you know Powershell you will have an easy time with Linux)
